README.md

```diff
@@ -112,7 +112,7 @@ In particular:
 - The `--debug` and `--stack` respectively allow for an error report generation and the stack of those errors. This is useful to understand and address any error that may occur the migration.
 - The `--remove-widgets` option to remove all widgets with the given keys during the migration.
   For example, suppose you have 200 ActiveUI 4 bookmarks in which you use Page Filters and/or Page Context Values.
```

Comment:
I don't think we're renaming ActiveUI 4, are we @Nouzbe ?

Reply:
I'll let @Nouzbe confirm. However with your changes, we're talking about:
- "Atoti UI Migration"
- "older versions of Atoti UI"
- "Migrating from Atoti UI 4 to 5"

which all refer to the version 4 of the product. Which I think is legit, because it's the same product, so it wouldn't make sense if it had different names. It's just a different version of the same product.
That said, I don't think we should rename anything in the `activeui-4` repo, just references to it in the 5+ repos.

Reply:
> I don't think we're renaming ActiveUI 4, are we @Nouzbe ?

That's the path we had started to follow indeed: not renaming it.

That being said, I tend to be convinced by @eliasinho 's comment here. If I am a customer and I am reading this, I guess I would get confused by this ActiveUI 4 to Atoti UI 5 thing here and I wouldn't understand that we are talking about different versions of the same product.

So I would actually prefer to also rename ActiveUI 4 to Atoti UI 4, in order to prevent confusion.